src/backend/tests/projects.test.ts

```diff
@@ -3,27 +3,78 @@
+  describe('all projects route', () => {
+    let responseObject: any;
+    let projectsResponse: Project[];
+
+    beforeEach(async () => {
+      const event = { path: '/projects', httpMethod: 'GET' };
+      responseObject = await handler(event, mockContext);
+      projectsResponse = JSON.parse(responseObject.body);
+    });
+
+    it('has 200 status code', () => {
+      expect(responseObject.statusCode).toBe(200);
+    });
+
+    it('has json headers', () => {
+      expect(responseObject.headers).toStrictEqual({ 'Content-Type': 'application/json' });
+    });
+
+    it('contains 5 projects', () => {
+      expect(projectsResponse.length).toBe(5);
+    });
+
+    it('has all required fields', () => {
+      projectsResponse.forEach((prj: Project) => {
+        expect(prj.hasOwnProperty('wbsNum')).toBeTruthy();
+        expect(prj.hasOwnProperty('name')).toBeTruthy();
+        expect(prj.hasOwnProperty('projectLead')).toBeTruthy();
+        expect(prj.hasOwnProperty('projectManager')).toBeTruthy();
+      });
+    });
 
-  it('has all required fields', () => {
-    exampleAllProjects.forEach((prj) => {
-      expect(prj.hasOwnProperty('wbsNum')).toBeTruthy();
-      expect(prj.hasOwnProperty('name')).toBeTruthy();
-      expect(prj.hasOwnProperty('projectLead')).toBeTruthy();
-      expect(prj.hasOwnProperty('projectManager')).toBeTruthy();
+    it('has proper project wbsNums', () => {
+      projectsResponse.forEach((prj: Project) => {
+        const project: Project = prj;
+        expect(project.wbsNum).toBeTruthy();
+        expect(project.wbsNum.workPackage).toEqual(0);
+      });
     });
   });
 
-  it('has proper project wbsNums', () => {
-    exampleAllProjects.forEach((prj) => {
-      const project: Project = prj;
-      expect(project.wbsNum).toBeTruthy();
-      expect(project.wbsNum.workPackage).toEqual(0);
+  describe('single project route', () => {
+    let responseObject: any;
+    let projectResponse: Project;
+
+    beforeEach(async () => {
+      const event = { path: '/projects/1.2.0', httpMethod: 'GET' };
+      responseObject = await handler(event, mockContext);
+      projectResponse = JSON.parse(responseObject.body);
+    });
+
+    it('has 200 status code', () => {
+      expect(responseObject.statusCode).toBe(200);
+    });
+
+    it('has json headers', () => {
+      expect(responseObject.headers).toStrictEqual({ 'Content-Type': 'application/json' });
+    });
+
+    it('has all required fields', () => {
+      expect(projectResponse.hasOwnProperty('wbsNum')).toBeTruthy();
+      expect(projectResponse.hasOwnProperty('name')).toBeTruthy();
+      expect(projectResponse.hasOwnProperty('projectLead')).toBeTruthy();
+      expect(projectResponse.hasOwnProperty('projectManager')).toBeTruthy();
+    });
+
+    it('has proper project wbsNums', () => {
+      expect(projectResponse.wbsNum).toBeTruthy();
+      expect(projectResponse.wbsNum.workPackage).toEqual(0);
     });
```

Comment:
Just to be thorough, adding a test case for when the wbs specified in the path does not exist could be helpful